### PR TITLE
(node/auxtel-dc01.cp) add el9 network config & tests

### PR DESCRIPTION
**Pull request by dtapiacl**

### CODE REVIEW

File: spec/hosts/nodes/auxtel-dc01.cp.lsst.org_spec.rb [hunk 1]
```diff
@@ -4,28 +4,50 @@
...
       let(:facts) do
-        lsst_override_facts(os_facts,
-                            is_virtual: false,
-                            virtual: 'physical',
-                            dmi: {
-                              'product' => {
-                                'name' => 'PowerEdge R630',
-                              },
-                            })
+        override_facts(os_facts,
```
Comment (jhoblitt) [inline spec/hosts/nodes/auxtel-dc01.cp.lsst.org_spec.rb]: Please use `lsst_override_facts` here.

File: spec/hosts/nodes/auxtel-dc01.cp.lsst.org_spec.rb [hunk 2]
```diff
@@ -4,28 +4,50 @@
...
+                       virtual: 'physical',
+                       dmi: {
+                         'product' => {
+                           'name' => 'PowerEdge R630',
+                         },
+                       })
       end
       let(:node_params) do
         {
           role: 'atsccs',
```
Comment (jhoblitt) [inline spec/hosts/nodes/auxtel-dc01.cp.lsst.org_spec.rb]: This role is incorrect... I have this fixed in another PR but this one will probably go in first.

**Review by jhoblitt — CHANGES_REQUESTED** [review]

File: spec/hosts/nodes/auxtel-dc01.cp.lsst.org_spec.rb (hunk 2 above)
Comment (dtapiacl) [reply spec/hosts/nodes/auxtel-dc01.cp.lsst.org_spec.rb]: Oh i just used the same role that was in foreman.

**Review by badenerb — APPROVED** [review]
LGTM

**Review by jhoblitt — APPROVED** [review]

**Pull request closed without merging**